### DISCUSSION TITLE
Qodana for Corda

### DISCUSSION THREAD

**Discussion by jsim77** [JetBrains/Qodana#220] [Q&A]
Hi,

Would like to ask, is there possible Corda to be use in Qodana?

If yes, do you have the tutorial and is it running under Testing or Mainnet?

Would appreciate your prompt response.

Thank you.

**Comment by brichbash** [comment]
Hello,

We don't have official support for Corda, and it isn't on our roadmap, as currently, we focus on languages/technologies already supported by JetBrains IDEs.
If there is a community plugin for IntelliJ with inspections specific to Corda, it's possible to use it in Qodana. Please check [this guide](https://www.jetbrains.com/help/qodana/qodana-yaml.html#Manage+plugins) on managing plugins in Qodana. Otherwise, we have [this section](https://www.jetbrains.com/help/qodana/extending-qodana-plugins.html#extending-qodana-developing-plugins) about developing custom plugins for IntelliJ which then can be used in Qodana.

**Reply by jsim77** [reply]
Thanks.
